config/deploy/production.rb

```diff
@@ -3,4 +3,4 @@
 # deploys to FRBM AWS production
 set :stage, :production
 set :rails_env, 'production'
-server '3.208.81.15', user: 'deploy', roles: [:web, :app, :db]
+server 'localhost', user: 'deploy', roles: [:web, :app, :db]
```

Comment:
the idea was this would only be used by ansible, which runs locally. it turns out someone already had that idea, which is why config/deploy/localhost.rb already exists. superseding changes moot this.